polkadot/node/core/pvf/src/testing.rs

```diff
@@ -59,21 +59,21 @@ pub fn validate_candidate(
 ///
 /// NOTE: This should only be called in dev code (tests, benchmarks) as it relies on the relative
 /// paths of the built workers.
-pub fn build_workers_and_get_paths(is_bench: bool) -> (PathBuf, PathBuf) {
+pub fn build_workers_and_get_paths() -> (PathBuf, PathBuf) {
 	// Only needs to be called once for the current process.
 	static WORKER_PATHS: OnceLock<Mutex<(PathBuf, PathBuf)>> = OnceLock::new();
 
-	fn build_workers(is_bench: bool) {
+	fn build_workers() {
 		let mut build_args = vec![
 			"build",
 			"--package=polkadot",
 			"--bin=polkadot-prepare-worker",
 			"--bin=polkadot-execute-worker",
 		];
-		if is_bench {
-			// Benches require --release. Regular tests are debug (no flag needed).
-			build_args.push("--release");
-		}
+
+		#[cfg(build_type = "release")]
```

Comment:
```suggestion
```

Reply:
To go this way, we would need something like

```rs
fn main() {
	if let Ok(mut profile) = std::env::var("PROFILE") {
		if profile == "debug" {
			profile = String::from("dev");
		}
		println!("cargo:rustc-env=BUILD_TYPE={}", profile);
	}
}
```

So maybe just keep it as is?

Reply:
Yeah fine. I was mainly wondering about the `production` profile, but this is probably just mapped to `release`.